src/test/java/features/GetUserFt.java

```diff
@@ -1,30 +0,0 @@
-package features;
-
-import io.cucumber.java.en.Given;
```

Comment:
This File was merged with `UpdateUserFt` to create the `UserFt` because of the use of common step definitions